x-pack/platform/plugins/shared/alerting/server/alerts_service/lib/retry_transient_es_errors.ts

```diff
@@ -29,15 +30,17 @@ export const retryTransientEsErrors = async <T>(
   {
     logger,
     attempt = 0,
+    additionalRetryableStatusCodes = [],
   }: {
     logger: Logger;
     attempt?: number;
+    additionalRetryableStatusCodes?: number[];
   }
 ): Promise<T> => {
   try {
     return await esCall();
   } catch (e) {
-    if (attempt < MAX_ATTEMPTS && isRetryableError(e)) {
+    if (attempt < MAX_ATTEMPTS && isRetryableError(e, additionalRetryableStatusCodes)) {
```

Comment:
are these changes used anywhere in this PR? if not, can we revert?

Reply:
https://github.com/elastic/kibana/commit/96d744f74413e78f92a7b8d2a4e63ccdb909d1cb